openpype/hosts/maya/plugins/publish/collect_look.py

```diff
@@ -25,6 +25,32 @@
 
 SHAPE_ATTRS = set(SHAPE_ATTRS)
 
+DEFAULT_FILE_NODES = frozenset(
+    ["file"]
+)
+
+ARNOLD_FILE_NODES = frozenset(
+    ["aiImage"]
+)
+
+REDSHIFT_FILE_NODES = frozenset(
+    ["RedshiftNormalMap"]
+)
+
+RENDERMAN_FILE_NODES = frozenset(
+    [
+        "PxrBump",
+        "PxrNormalMap",
+        # PxrMultiTexture (need to handle multiple filename0 attrs)
+        "PxrPtexture",
+        "PxrTexture",
+    ]
+)
```

Comment:
Would it make more sense to store it like this:
```python
file_nodes = {
    # Maya
    "file": "filename",

    # Arnold
    "aiImage": "filename",

    # Redshift
    "RedshiftNormalMap": "tex0",

    # Renderman
    "PxrBump": "filename",
    "PxrNormalMap": "filename",
    "PxrPtexture": "filename",
    "PxrTexture": "filename",
    # PrxMultiTexture is a special case with multiple filename attributes
    # So this could maybe be a special function?
    "PxrMultiTexture": get_pixar_multi_texture
}

# Example usages
node_type = cmds.nodeType(node)
attr = file_nodes[node_type]
path = cmds.getAttr('{}.{}'.format(node, attr))
```

Special edge case:
```python
def get_pixar_multi_texture(node):
    filepaths = []
    for path in path_attributes:
        filepaths.append(path)
    return filepaths
```

Somehow feels more manageable overall instead of having the logic spread out as much as it is now?